packages/data-point/lib/reducer-list/resolve.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
```

Comment:
is this new code or should had been a git mv?

Reply:
should have been a git mv